ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -140,25 +142,33 @@
   private final KsqlEngine ksqlEngine;
   private final ReplayableCommandQueue replayableCommandQueue;
   private final long distributedCommandResponseTimeout;
+  private final ActiveChecker activeChecker;
+  private final TimeUtils timeUtils;
```

Comment:
Could I request that we don't use `TimeUtils` for handling time please? It's a pretty horrid class.  Use the `java.time.Clock` interface.